packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -155,7 +157,7 @@ const WalletCashierHeader: React.FC<TProps> = ({ hideWalletDetails }) => {
                                     size='sm'
                                     weight={isActiveTab ? 'bold' : 'normal'}
                                 >
-                                    {tab.text}
+                                    {t(tab.text)}
```

Comment:
This wont work during string extraction btw. When you try to extract this using the regex, it will capture the variable `tab.text`instead of the actual text, because during build time, nothing is evaluated

Reply:
In that case we need to translate in the constants itself right? Means we need to either:
- move the constants inside the component / treated it as hook, or
- Create a wrapper like `<Localize /> ` component